crates/forge/src/test_filter.rs

```diff
@@ -124,25 +124,25 @@ impl TestCaseFilter for TestsFilter {
 #[cfg(test)]
 mod tests {
     use crate::test_filter::TestsFilter;
-    use cairo_lang_sierra::program::{Program, ProgramArtifact, Version, VersionedProgram};
+    use cairo_lang_sierra::program::Program;
     use forge_runner::expected_result::ExpectedTestResult;
+    use forge_runner::package_tests::raw::{DebugInfo, ProgramArtifact};
     use forge_runner::package_tests::with_config_resolved::{
         TestCaseResolvedConfig, TestCaseWithResolvedConfig, TestTargetWithResolvedConfig,
     };
     use forge_runner::package_tests::{TestDetails, TestTargetLocation};
-
-    fn program_for_testing() -> VersionedProgram {
-        VersionedProgram::V1 {
-            version: Version::<1>,
-            program: ProgramArtifact {
-                program: Program {
-                    type_declarations: vec![],
-                    libfunc_declarations: vec![],
-                    statements: vec![],
-                    funcs: vec![],
-                },
-                debug_info: None,
+    use std::sync::Arc;
+    use universal_sierra_compiler_api::compile_sierra_to_casm;
+
+    fn program_for_testing() -> ProgramArtifact {
+        ProgramArtifact {
+            program: Program {
+                type_declarations: vec![],
+                libfunc_declarations: vec![],
+                statements: vec![],
+                funcs: vec![],
             },
+            debug_info: DebugInfo::default(),
```

Comment:
Why default instead of None now?

Reply:
changed https://github.com/foundry-rs/starknet-foundry/pull/2264